tests/modules/text_field_embedders/custom_feature_embedder.py

```diff
@@ -0,0 +1,50 @@
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.data import Token, Vocabulary, Dataset, Instance
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer, FeatureIndexer
+import numpy as np
+from allennlp.modules.text_field_embedders import BasicTextFieldEmbedder
+from allennlp.modules.token_embedders import Embedding
+from allennlp.modules.token_embedders.bypass_embedder import BypassEmbedder
+from allennlp.nn.util import arrays_to_variables
+
+
+def lowercase_feature_extractor(token: Token):
+    if token.text[0].isupper():
+        return np.array([1.0], dtype=np.float32)
+    else:
+        return np.array([0.0], dtype=np.float32)
+
+
+
+class TestCustomFeatureFieldEmbedder(AllenNlpTestCase):
```

Comment:
This should be `TestBypassEmbedder(AllenNlpTestCase)`, and the name of the file should match.

Reply:
Ah, I see that you have a separate file for that.  I'd probably just put these tests in that file.  Though, actually, these tests are probably unnecessary, as this is just an integration test on all of the components of the featurized token indexer / embedder.  We have plenty of integration tests for this pipeline, so as long as the components are tested well enough, an additional integration test isn't necessary.